# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/websocket/moz.build
# Commit: f7b6aac1b58b
# Full Hash: f7b6aac1b58bc959e431136e1b23231abe55d0b1
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-07-16 21:27:37
# Regressor Bug: 1497249
# File Overlap Count: 4
# Description:
#   Bug 1497249 - P2: ipdl for nsIWebSocketConnection r=michal
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30624
# ==============================================================================

diff -r 74bb3c2ab767 -r f7b6aac1b58b netwerk/protocol/websocket/moz.build
--- a/netwerk/protocol/websocket/moz.build	Thu Jul 16 12:59:23 2020 +0000
+++ b/netwerk/protocol/websocket/moz.build	Thu Jul 16 13:34:58 2020 +0000
@@ -23,6 +23,8 @@
     'WebSocketChannel.h',
     'WebSocketChannelChild.h',
     'WebSocketChannelParent.h',
+    'WebSocketConnectionChild.h',
+    'WebSocketConnectionParent.h',
     'WebSocketEventListenerChild.h',
     'WebSocketEventListenerParent.h',
     'WebSocketEventService.h',
@@ -36,6 +38,8 @@
     'WebSocketChannel.cpp',
     'WebSocketChannelChild.cpp',
     'WebSocketChannelParent.cpp',
+    'WebSocketConnectionChild.cpp',
+    'WebSocketConnectionParent.cpp',
     'WebSocketEventListenerChild.cpp',
     'WebSocketEventListenerParent.cpp',
     'WebSocketEventService.cpp',
@@ -45,6 +49,7 @@
 IPDL_SOURCES += [
     'PTransportProvider.ipdl',
     'PWebSocket.ipdl',
+    'PWebSocketConnection.ipdl',
     'PWebSocketEventListener.ipdl',
 ]
 